comms/peer/src/Peer.ts

```diff
@@ -1120,7 +1278,7 @@ export class Peer implements IPeer {
             break;
           }
 
-          if (this.httpClient.lighthouseUrl !== payload.lighthouseUrl || this.currentLayer !== payload.layer) {
+          if (this.currentLayer !== payload.layer) {
```

Comment:
Why should we omit the lighthouse URL check?

Reply:
Ufff. Nice catch. I actually had changed this for a test in my local network :facepalm: